src/server/service/kv.rs

```diff
@@ -1238,6 +1239,13 @@ fn extract_region_error<T>(res: &storage::Result<T>) -> Option<RegionError> {
             err.set_server_is_busy(server_is_busy_err);
             Some(err)
         }
+        Err(Error::GCWorkerTooBusy) => {
+            let mut err = RegionError::new();
```

Comment:
do we have a metric to record it in TiKV?

Reply:
Added at gc_worker.rs L248

Reply:
Also please remember to update our grafana dashboard to utilize these new metrics.